tools/mason/MasonUpdate.chpl

```diff
@@ -66,12 +66,15 @@ proc UpdateLock(args: list(string), tf="Mason.toml", lf="Mason.lock") {
     const tomlPath = projectHome + "/" + tf;
     const lockPath = projectHome + "/" + lf;
 
-
     updateRegistry(tf, args);
     const openFile = openreader(tomlPath);
     const TomlFile = parseToml(openFile);
+    if isDir(SPACK_ROOT) {
```

Comment:
@ankingcodes - Does this catch any cases that are not already caught in `MasonBuild`?

If we keep this check, we need to update the error message.

When upgrading from the previous version of `mason` and doing a `mason build`, we get this error message:

```
Mason could not find Spack at $MASON_HOME.
To install Spack, use : mason external --setup
```

where the other error message is probably more fitting:

```
Mason has been updated. To use mason external, call : mason external --setup
```

Reply:
I'm in favour of keeping this check here. I'll update the error message.

Reply:
OK, will this error message fire if a user is building (and updating) a project that does not use any external packages?

Reply:
Yes, if an user has a SPACK_ROOT that hasn't been updated.

Reply:
I think we only want this error message to fire if the user is doing anything that would require external packages or `mason external` commands. That way, a user that never ran `mason external --setup` before will never run into any of these errors.

Reply:
I'll add an additional check for checking 'external' in TOML file.